examples/Tables/Table.md

```diff
@@ -171,7 +171,7 @@ import {
 } from 'mark-one';
 
 <Table>
-  <TableHead>
+  <TableHead isSticky={true}>
```

Comment:
For a more advanced feature like this, I think it would be good to break out a separate section of the documentation that explains a little more about the situations where you would want to use `isSticky`, rather than just add it into one of the other examples. You could also create an example of a very long table to show how it works in practice (You can always an array iterator to generate a bunch of rows automatically).